worker/send_sms.js

```diff
@@ -1,17 +1,45 @@
-module.exports = function(notifier_messager) {
+module.exports = function(notifier_messager, rateLimiter) {
   var LUMBERYARD_EVENT = "send_sms";
 
-  return function(id, event, cb) {
-    if (event.event_type !== "send_sms") {
-      return process.nextTick(cb);
-    }
+  // limit one SMS/number/5 minutes
+  var TTL = 1000 * 60 * 5;
```

Comment:
This rate limit only applies to sending the install link to a mobile device via SMS. It won't affect the publishing/sharing process.